internal/executor/instance/persistentworker/isolation/tart/tart.go

```diff
@@ -151,9 +152,20 @@ func (tart *Tart) Run(ctx context.Context, config *runconfig.RunConfig) (err err
 		})
 	}
 
-	return remoteagent.WaitForAgent(ctx, tart.logger, ip,
+	addTartListBreadcrumb()
+	addDHCPDLeasesBreadcrumb()
+
+	err = remoteagent.WaitForAgent(ctx, tart.logger, ip,
 		tart.sshUser, tart.sshPassword, "darwin", "arm64",
 		config, true, hooks, preCreatedWorkingDir)
+	if err != nil {
+		addTartListBreadcrumb()
+		addDHCPDLeasesBreadcrumb()
```

Comment:
I think this will permanently add the breadcrumbs. Let's just report a single event here with the breadcrumbs?

Reply:
>I think this will permanently add the breadcrumbs.

What do you mean by "permanently"? The breadcrumbs will only be included if an exception occurs and will be attached to that exception.

Reply:
The persistent worker process is running constantly and executes different tasks. In this situation the next tasks will have breadcrumbs of the previous task.

Reply:
Ohh, good catch! Fixed in https://github.com/cirruslabs/cirrus-cli/pull/634/commits/82549d82c7d87bbdd3099f177a3b8b1502b9775a as per https://docs.sentry.io/platforms/go/concurrency/.